src/main/java/org/jahia/modules/contenteditor/api/forms/impl/EditorFormServiceImpl.java

```diff
@@ -564,19 +564,13 @@ private EditorFormField generateEditorFormField(ExtendedItemDefinition itemDefin
         if (selectorOptions != null && !selectorOptions.isEmpty()) {
             Map<String, ChoiceListInitializer> initializers = choiceListInitializerService.getInitializers();
 
-            Map<String, String> replacements = new HashMap<>();
-            if (existingNode != null) {
-                replacements.put("$currentSite", existingNode.getResolveSite().getPath());
-            }
-
             Map<String, Object> context = new HashMap<>();
             context.put("contextType", primaryNodeType);
             context.put("contextNode", existingNode);
-            context.put("contextParent", parentNode);
```

Comment:
are we sure it's safe to remove "contextParent"? Asking since I'm seeing a lot of references to `context.get("contextParent")` in core and not sure if all of them are handling null cases of this properly or what other impacts removing this has. @dgriffon any thoughts on this?

Reply:
There is "fallback" strategy in choicelist initializer to use contextNode in case parentNode is not available, in fact in most initializers where this is used contextNode is tried first and contextParent is fallback (I looked at all the ones in the core).